### ISSUE TITLE
[Bug] General fixes for three Perl scripts are needed

### ISSUE THREAD

**Issue by wenzeslaus** [OSGeo/grass#1360]
**Describe the bug**

Perl::Critic (perlcritic) included in Mega-Linter identified the following errors in `display/d.text/test.pl` and `utils/copywrite.pl`.

```
Results of perlcritic linter (version 1.138)
See documentation on https://nvuillam.github.io/mega-linter/descriptors/perl_perlcritic/
-----------------------------------------------

❌ [ERROR] display/d.text/test.pl
    Code before strictures are enabled at line 2, column 1.  See page 429 of PBP.  (Severity: 5)
    Bareword file handle opened at line 52, column 1.  See pages 202,204 of PBP.  (Severity: 5)
    Two-argument "open" used at line 52, column 1.  See page 207 of PBP.  (Severity: 5)

✅ [SUCCESS] general/g.parser/test.pl
❌ [ERROR] utils/copywrite.pl
    Code before strictures are enabled at line 28, column 1.  See page 429 of PBP.  (Severity: 5)
    Bareword file handle opened at line 76, column 32.  See pages 202,204 of PBP.  (Severity: 5)
    Two-argument "open" used at line 76, column 32.  See page 207 of PBP.  (Severity: 5)
    Bareword file handle opened at line 98, column 2.  See pages 202,204 of PBP.  (Severity: 5)
    Two-argument "open" used at line 98, column 2.  See page 207 of PBP.  (Severity: 5)
    Expression form of "eval" at line 146, column 12.  See page 161 of PBP.  (Severity: 5)
    Bareword file handle opened at line 152, column 5.  See pages 202,204 of PBP.  (Severity: 5)
    Two-argument "open" used at line 152, column 5.  See page 207 of PBP.  (Severity: 5)
```

Running perlcritic manually reveals additional issues with file `vector/v.clean/test/v.rand.poly`.

```
vector/v.clean/test/v.rand.poly: Code before strictures are enabled at line 5, column 1.  See page 429 of PBP.  (Severity: 5)
vector/v.clean/test/v.rand.poly: Bareword file handle opened at line 47, column 1.  See pages 202,204 of PBP.  (Severity: 5)
vector/v.clean/test/v.rand.poly: Two-argument "open" used at line 47, column 1.  See page 207 of PBP.  (Severity: 5)
```

**To Reproduce**

```
perlcritic .
```

**Expected behavior**
No errors reported.

Fixing code, rewriting to Python, removing unused file, or ignoring the errors explicitly ([bending rules](https://metacpan.org/pod/distribution/Perl-Critic/bin/perlcritic#BENDING-THE-RULES)) is all fine.

**System description (please complete the following information):**

- master branch (7.9) with #766 merged

**Additional context**

For example, #766 is a rewrite of some other Perl scripts to Python.

**Comment by ocsmit** [comment]
I am going to look at rewriting these scripts to Python 3, unless of course there has already been progress made elsewhere in solving this.

**Comment by wenzeslaus** [comment]
Thank you. It's yours!

**Comment by wenzeslaus** [comment]
`perlcritic .` reports OK. Great.

**Comment by wenzeslaus** [comment]
Fixed by #1493. Basic Perl checks (not perlcritic) enabled in #1493. (Python checks are already in place.)
